pkg/operator/assets/bindata.go

```diff
@@ -1594,6 +1599,21 @@ data:
       - "failed to initialize the cluster: Cluster operator monitoring is still updating"
       installFailingReason: MonitoringOperatorStillUpdating
       installFailingMessage: Timeout waiting for the monitoring operator to become ready
+    - name: AuthenticationOperatorDegraded
+      searchRegexStrings:
+      - "Cluster operator authentication Degraded is True"
+      installFailingReason: AuthenticationOperatorDegraded
+      installFailingMessage: Timeout waiting for the authentication operator to become ready
+    - name: GeneralOperatorDegraded
+      searchRegexStrings:
+      - "Cluster operator.*Degraded is True"
+      installFailingReason: GeneralOperatorDegraded
+      installFailingMessage: Timeout waiting for an authentication operator to become ready
```

Comment:
This one is a general message for operators other than authentication one, if degraded...